packages/server/graphql/public/typeDefs/UserLogInPayload.graphql

```diff
@@ -7,12 +7,12 @@ type UserLogInPayload {
   """
   The new auth token
   """
-  authToken: ID
-  userId: ID
+  authToken: ID!
+  userId: ID!
   """
   if a new user is created
   """
-  isNewUser: Boolean
+  isNewUser: Boolean!
```

Comment:
-1 This is not correct, we're returning null here on login error: https://github.com/ParabolInc/parabol/blob/62792bda08c4a1fb56200944569513445cc3aeae/packages/server/graphql/public/mutations/loginWithPassword.ts#L31